docs/route-tasks.md

```diff
@@ -0,0 +1,238 @@
+    ]
+}>
+
+<TabItem value="java">
+
+[Sample source reference](https://github.com/temporalio/java-samples/blob/master/src/main/java/io/temporal/samples/hello/HelloActivity.java#L91-L99)
+```java
+// Worker is created and subscribed to the "hello-world" Task Queue 
+Worker worker = factory.newWorker("hello-world");
+// Worker is registered to handle helloworld Workflow
+worker.registerWorkflowImplementationTypes(HelloWorldWorkflowImpl.class)
+// Worker is registered to handle helloworld Activity
+worker.registerActivityImplementationTypes(HelloWorldActivityImpl)
+```
+
+</TabItem>
+<TabItem value="go">
+
+[Sample source reference](https://github.com/temporalio/go-samples/blob/master/helloworld/worker/main.go#L20-L23)
+```go
+// Worker is created and subscribed to the "hello-world" Task Queue
+w := worker.New(c, "hello-world", worker.Options{})
+defer w.Stop()
+// Worker is registered to handle the helloworld Workflow
+w.RegisterWorkflow(helloworld.Workflow)
+// Worker is registered to handle the helloworld Activity
+w.RegisterActivity(helloworld.Activity)
+```
+
+</TabItem>
+</Tabs>
+
+In the next code snippet, we configure the "Hello World" Workflow to send its Tasks to the "hello-world" Task Queue and then start its execution:
+
+<Tabs
+    defaultValue="go"
+    values={[
+        { label: 'Java', value: 'java'},
+        { label: 'Go', value: 'go' },
+    ]           
+}>
+
+<TabItem value="java">
+
+[Sample source reference](https://github.com/temporalio/java-samples/blob/master/src/main/java/io/temporal/samples/hello/HelloActivity.java#L91-L99)
+```java
+HelloWorldWorkflow workflow =
+    client.newWorkflowStub(
+        HelloWorldWorkflow.class,
+        // Workflow Options specify the "hello-world"" Task Queue
+        // Which is the Task Queue that the Workflow will send its Tasks to
+        WorkflowOptions.newBuilder().setTaskQueue("hello_world").build());
+// Workflow is executed
+WorkflowExecution workflowExecution =
+    WorkflowClient.start(workflow::helloWorld);
+```
+
+</TabItem>
+<TabItem value="go">
+
+[Sample source reference](https://github.com/temporalio/go-samples/blob/master/helloworld/starter/main.go#L20-L25)
+```go
+// Workflow Options specify the "hello-world" Task Queue
+// Which is the Task Queue that the Workflow will send its Tasks to
+workflowOptions := client.StartWorkflowOptions{
+  ID:        "hello_world_workflowID",
+  TaskQueue: "hello-world",
+}
+// Workflow Options are passed to the execution of the Workflow
+we, err := c.ExecuteWorkflow(
+  context.Background(), 
+  workflowOptions, 
+  helloworld.Workflow, 
+  "Temporal",
+)
+```
+
+</TabItem>
+</Tabs>
+
+:::caution
+
+If a Worker pulls a Task that it has not been registered to execute, an "unknown workflow type" error will occur.
+
+:::
+
+The above examples illustrate the fundamental approach to pairing Tasks with Workers. In the next section we look at a more complex scenario involving Activities with host-specific file dependencies.
+
```

Comment:
Host specific task routing is a very important use case. But the more important use case is running different activities and workflows in different services (which are essentially pools of processes). I think we want to have a section on using task queues to route to different service/deployment.